articles/libraries/lock-android/v1/internationalization.md

```diff
@@ -0,0 +1,37 @@
+---
+section: libraries
+toc_title: Internationalization
```

Comment:
Can you tell me what this metadata does? I have seen it in some docs, mainly about libraries, (without the `toc` metadata, like this one https://github.com/auth0/docs/blob/master/articles/libraries/lock-ios/v1/logging.md) but I don't get what it does.

Reply:
It was for the sort of "related docs" section used on some older docs - see the `/libraries/lock-android/v1/index` - that's the only place I see still using it, while I'm in here I think I'll remove that, add a list of "related topics" or next steps, and we can strip that metadata item out anywhere it is.